lab_5_scrapper/scrapper.py

```diff
@@ -168,14 +298,43 @@ def _fill_article_with_text(self, article_soup: BeautifulSoup) -> None:
         Args:
             article_soup (bs4.BeautifulSoup): BeautifulSoup instance
         """
+        all_body = article_soup.find_all('div')
+
+        texts = []
+        if all_body:
+            all_divs = all_body[0].find_all('div', class_='entry-content entry clearfix')
+            texts = []
+            for div_bs in all_divs:
+                texts.append(div_bs.text)
+
+        self.article.text = ''.join(texts)
 
-    def _fill_article_with_meta_information(self, article_soup: BeautifulSoup) -> None:
+    def _fill_article_with_meta_information(self, article_soup: BeautifulSoup) -> Article:
```

Comment:
you should not modify interface - article should be available as a class field from any Parser method

Reply:
done :(